Formula/rke.rb

```diff
@@ -0,0 +1,21 @@
+class Rke < Formula
+  desc "Rancher Kubernetes Engine, a Kubernetes installer that works everywhere"
+  homepage "https://rancher.com/docs/rke/v0.1.x/en/"
+  url "https://github.com/rancher/rke.git",
+    :tag => "v0.1.9"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/rancher/rke/").install Dir["*"]
+    system "go", "build", "-ldflags",
+           "-w -X main.VERSION=#{version}",
+           "-o", "#{bin}/rke",
+           "-v", "github.com/rancher/rke/"
+  end
+
+  test do
+    system bin/"rke", "--version"
```

Comment:
We need a test that exercises the some of the functionality of the app.

Reply:
I updated the test to generate a template cluster.yml configuration file and assert the file was created.